src/dispatch/plugins/dispatch_slack/case/interactive.py

```diff
@@ -662,9 +658,9 @@ def _create_snooze_filter(
             # Get the existing filters for the signal
             _create_snooze_filter(
                 db_session=db_session,
-                form_data=metadata["form_data"],
+                form_data=context["subject"].form_data,
                 user=user,
-                subject=subject,
+                subject=context["subject"],
```

Comment:
nit: could the `form_data` arg be removed in as the `subjet` arg would also have the form_data values/

Reply:
yeah, good catch. i'll update in a new pr.